python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -393,11 +395,9 @@ def __init__(
         self.executionGraphFixup = executionGraphFixup
         self.report: Report | None = None
 
-        # We set default start method as spawn for MacOS and fork for Linux;
-        # None for all other platforms to use multiprocessing default.
```

Comment:
Is fork the multiprocessing default then?

Reply:
Fork is default for Linux, spawn is default for MacOS and Windows.